plugins/package-managers/node/src/main/kotlin/NpmSupport.kt

```diff
@@ -137,14 +137,13 @@ internal fun parseVcsInfo(packageJson: PackageJson): VcsInfo {
     )
 }
 
+typealias PackageDetailsFunction = (packageName: String) -> PackageJson?
```

Comment:
`Function` is so generic, how about `PackageDetailsProvider` or `GetPackageDetails`?

Reply:
I had "Provider" initially, but compared to our other providers, like `PackageCurationProvider`, this is way more lightweight, which is why I went for "Function".

Also I didn't want to call it "GetPackageDetails", as that name IMO better fits for the variable / parameter name in line 146, and I didn't want to call the parameter and type basically the same, just with different capitalization.

Reply:
I think it would not be bad to call it the same. But it's also not so important.

Reply:
Hmm, as somewhat of a compromise, would you like `GetPackageDetailsFun` better?

Reply:
Let's eventually change this later on.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/9738.